website/docs/glue-catalog.md

```diff
@@ -58,7 +57,7 @@ datasets:
   -
     tableBasePath: s3://path/to/source/data
     tableName: table_name
-    partitionSpec: partitionpath:VALUE
+    partitionSpec: partitionpath:VALUE # you don't need to specify partitionSpec for ICEBERG sourceFormat 
```

Comment:
`# you only need to specify partitionSpec for HUDI sourceFormat`

Reply:
done